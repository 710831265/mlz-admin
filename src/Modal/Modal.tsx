import React, { useState, useContext } from 'react';
import { Modal as AntdModal } from 'antd';

const Modal = AntdModal;

export default Modal;
