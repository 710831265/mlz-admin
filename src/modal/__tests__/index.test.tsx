import React, { Component } from 'react';
import { testMount, testSnapshot } from '../../../tests';
import Modal from '..';

describe('ðŸ§ª Modal', () => {
  testMount(Modal);
  testSnapshot(Modal);
});
