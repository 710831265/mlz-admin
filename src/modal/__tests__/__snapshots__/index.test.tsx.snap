// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ðŸ§ª Modal snapshot test snapshot comparing 1`] = `""`;
